webapp/src/components/AssetPage/OwnersTable/OwnersTable.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { useEffect, useState } from 'react'
+import { Table, Loader, Icon, Row, Pagination } from 'decentraland-ui'
+import { t } from 'decentraland-dapps/dist/modules/translation/utils'
+import { Link } from 'react-router-dom'
+import { locations } from '../../../modules/routing/locations'
+import { nftAPI } from '../../../modules/vendor/decentraland'
+import { LinkedProfile } from '../../LinkedProfile'
+import ListedBadge from '../../ListedBadge'
+import {
+  OrderDirection,
+  OwnersFilters,
+  OwnersResponse,
+  OwnersSortBy,
+  Props
+} from './OwnersTable.types'
+import styles from './OwnersTable.module.css'
+
+const ROWS_PER_PAGE = 6
+
+const OwnersTable = (props: Props) => {
+  const { asset, orderDirection = OrderDirection.ASC } = props
+
+  const [owners, setOwners] = useState<OwnersResponse[]>([])
+  const [total, setTotal] = useState(0)
+  const [page, setPage] = useState(1)
+  const [totalPages, setTotalPages] = useState(1)
```

Comment:
```suggestion
  const [totalPages, setTotalPages] = useState()
```
I believe this should start as `undefined` or `null` until we know how many pages there are, wdyt?